frontend/.env.development

```
@@ -0,0 +1,2 @@
+VITE_PUBLIC_URL=http://localhost:5173
+VITE_SITE_TITLE=Champagne Festival (Dev)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)